### PR TITLE
chore: create basic setup for new landingpage

### LINKED ISSUE #757: Setup new Landingpage
Setup the new landingpage according to the design style guides. The landingpage should contain

* the navigation
* header
* footer

The landingpage should be set up as subpage of the current website. So it must not be hidden but we should change the landingpage in the end in one step.
Think about wether a new package should be created.

### PR DESCRIPTION
**Pull request by janivo**
closes #757

### CODE REVIEW

**Review by MariaLStefan — CHANGES_REQUESTED** [review]
Looks great! Thanks for setting up the new landingpage :) I only found some minor things that could be adjusted.

File: packages/landingpage-new/styles/Home.module.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
+.container {
+  padding: 0 2rem;
+}
```
[the remark this answers is not in the document]
Comment (janivo) [reply packages/landingpage-new/styles/Home.module.css:3]: These were some leftovers of the initial app created by next. I removed them for clarity

File: packages/landingpage-new/components/layout/header.module.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  background-color: white;
+  height: 120px;
+  display: grid;
+  grid-template-columns: auto 1fr auto auto;
+  grid-template-rows: 1fr;
+  column-gap: 30px;
+  grid-template-areas:
+    "logo navbar github inovex";
+  align-items: center;
+}
```
[the remark this answers is not in the document]
Comment (janivo) [reply packages/landingpage-new/components/layout/header.module.scss]: nice idea!

File: packages/landingpage-new/components/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from 'react';
+import styles from './layout.module.scss';
+import Header from './layout/header';
+import Footer from "./layout/footer";
```
[the remark this answers is not in the document]
Comment (janivo) [reply packages/landingpage-new/components/layout.tsx]: Good spot. I added an ESLint Rule to make sure

**Pull request merged**